### PR TITLE
Basic Delete All and Reschedule All functionality

### PR DESCRIPTION
**Pull request by eronisko**
This is related to #4. Allows user to re-schedule or delete all jobs on the scheduled or failed list.

image

### CODE REVIEW

**Comment by eronisko** [comment]
Hi @statianzo, any thoughts on this?

File: web/views/scheduled.erb [hunk 1]
```diff
@@ -48,7 +48,23 @@
...
+      </form>
+      &nbsp;
+      <form action="<%= link_to "jobs" %>" method="post" class="form-inline"
+            onsubmit="return confirm('Are you sure you want to delete all <%= @list.total %> jobs?');">
+        <input type="hidden" name="_method" value="delete" />
+        <input type="hidden" name="scope" value="scheduled" />
+        <button class="button small alert" title="Delete All"><i class="fa fa-trash"></i> Delete All</button>
+      </form>
+    </div>
+  <% end %>
```
Comment (statianzo) [inline web/views/scheduled.erb]: This duplicates the delete/retry buttons of `failing.erb`. Create a shared template and call it via `<%== erb :_all_job_actions %>`

File: web/views/scheduled.erb [hunk 2]
```diff
@@ -48,7 +48,23 @@
...
+  <% if @list.total > 0 %>
+    <div class="small-4 columns text-right">
+      <form action="<%= link_to "jobs" %>" method="post" class="form-inline">
+        <input type="hidden" name="_method" value="put" />
+        <input type="hidden" name="scope" value="scheduled" />
+        <button class="button small" title="Run All Immediately"><i class="fa fa-play-circle"></i> Run All</button>
+      </form>
+      &nbsp;
+      <form action="<%= link_to "jobs" %>" method="post" class="form-inline"
+            onsubmit="return confirm('Are you sure you want to delete all <%= @list.total %> jobs?');">
```
Comment (statianzo) [inline web/views/scheduled.erb]: Good call on the `confirm`. Run All Immediately should also have a confirm.

File: web/views/scheduled.erb (hunk 1 above)
Comment (eronisko) [reply web/views/scheduled.erb]: A partial template was the original plan, but the icons and action names are actually not the same (`Retry All` vs `Run All`). 

Compare `failing.erb`
image

and `scheduled.erb`
image

Do you want to unify this somehow?

**Comment by statianzo** [comment]
Published as `0.7.0`. Thank you for the contribution!

**Pull request merged**